spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -87,7 +87,7 @@ <h2 th:text="#{admin.api-keys-consumer.subtitle}">[Post Administration</h2>
                                 </thead>
                                 <tbody>
                                     <tr th:each="apiKey : ${apiKeys}" th:id="|@{item}${apiKey.id}|">
-                                        <td class="a-center " th:text=" ${apiKey.serviceApiName != null} ? ${apiKey.serviceApiName} : '' ">Service API name</td>
+                                        <td class="a-center " th:text=" ${apiKey.serviceApiName != null} ? ${apiKey.serviceApiName + ' / ' + apiKey.serviceApi.uriIdentifier} : '' ">Service API name</td>
```

Comment:
General topic to be discussed: the ID will often be generated by the tutorial, so that at first the users will not recognize this ID. Therefore it should probably be displayed as "side note" and including an explanation, e.g. we could use the approach `Service Name (ID: service-identifier)` or maybe `Service Name (ID/path: /service-identifier)` .

Reply:
@ChristophBaierATB 

Summary, as discussed: the best approach would be to unify the approach in all places, using `Service Name (Identifier)` as column heading and add the CoatRack service identifier in brackets (as additional information).

Exception to this rule: in the "Edit Services" section, the CoatRack service identifier should stay as separate column, as it is related to a separate input field.